src/nidigital/nidigital.mak

```diff
@@ -8,5 +8,9 @@ MODULE_FILES_TO_COPY := $(DEFAULT_PY_FILES_TO_COPY)
 
 RST_FILES_TO_GENERATE := $(DEFAULT_RST_FILES_TO_GENERATE)
 
+# Add custom types to copy
```

Comment:
Comment is not very useful. More useful would be to say why this needs to be copied and/or where to.
I assume you copied this from `niscope` so whatever you do do it in both locations - remove / leave alone / reword

Reply:
I removed it. None of the other rules in the file have any comments.